### DISCUSSION TITLE
Is it possible to combine WEC-SIM and OPENFAST to simulate floating wind-wave cogeneration plants?

### DISCUSSION THREAD

**Discussion by tatesnow** [WEC-Sim/WEC-Sim#1021] [Q&A]
Is it possible to combine WEC-SIM and OPENFAST to simulate floating cogeneration plants? If possible, is there any recommended way?

**Comment by akeeste** [comment]
Hello @tatesnow. Sorry for the late reply, we typically don't use the Discussion board. The development team closely monitors the issue board instead. I'll close the Discussion tab so that this is clear. 

To answer your question, I would refer you to the recent issue #989. If you have additional questions after reviewing that thread, I encourage you to open a new issue and continue the discussion there

**Comment by kmruehl** [comment]
@tatesnow we just re-opened the discussion board, and I noticed that your question relates to a recent publication (which you may already be aware of or be involved with). 

A coupling framework between OpenFAST and WEC-Sim. Part I: Validation and dynamic response analysis of IEA-15-MW-UMaine FOWT
https://www.sciencedirect.com/science/article/pii/S0960148124003148
